src/Workspaces/CSharpTest/OrganizeImports/OrganizeUsingsTests.cs

```diff
@@ -23,12 +20,14 @@ public class OrganizeUsingsTests
             bool separateImportGroups = false,
             CSharpParseOptions options = null)
         {
-            using var workspace = TestWorkspace.CreateCSharp(initial);
-            var document = workspace.CurrentSolution.GetDocument(workspace.Documents.First().Id);
+            using var workspace = new AdhocWorkspace();
```

Comment:
❔ Why did this change? `AdhocWorkspace` constructed in this manner can be problematic in tests, so we probably should not be using it.

Reply:
➡️ The problems with this are avoided by the s_CreationHook we're using.